examples/lib/stories/utils/particles.dart

```diff
@@ -157,7 +156,7 @@ class ParticlesGame extends BaseGame {
     return Particle.generate(
       count: 5,
       generator: (i) => MovingParticle(
-        to: randomCellOffset() * .5,
+        to: randomCellVector2() * .5,
```

Comment:
These will create two Vector2 objects, you can mutate the one created with the random function with ..scale(.5)